src/github.com/getlantern/flashlight/genconfig/genconfig.go

```diff
@@ -111,15 +111,15 @@ func main() {
 		}
 	}
 	if *proxiedSitesOutFile != "" {
-		proxiedSitesTmpl := loadTemplate(*proxiedSitesOutFile)
+		proxiedSitesTmpl := loadTemplate("proxiedsites.go.tmpl")
```

Comment:
Actually @aranhoide this one is interesting -- I wonder if that could be causing any issues on the server side generation? I don't think any go files are used there, but maybe worth a look.